README.md

```diff
@@ -74,7 +74,7 @@ preferred development workflow.
 
 ### Prerequisites
 
-First, ensure you have podman-compose, podman and java 11 installed:
+First, ensure you have podman-compose, podman and java 17 installed:
```

Comment:
This section also mentions docker and docker-compose. This didn't work for me and I think we should remove all the mentions to docker (since we should go only for podman). wdyt?

Reply:
Yeah, I think that's fine. FWIW, you can probably get docker to work by manually setting `DOCKER_HOST` (the gradle setup uses `DOCKER_HOST` as-is if specified, otherwise assumes rootless podman for the current user). I think it's better to standardize the setup on podman/podman-compose though.

Reply:
Remove all the mentions to docker